common/journal_entries/imperia_true_neutral_journal_entries.txt

```diff
@@ -19,12 +19,15 @@
 			# Kinda dirty fix, since we cannot check for the existance of dynamic, non-specific countries
 			if = {
 				limit = {
-					local_var:true_neutral_applicant_signee_list_counter < imperia_true_neutral_remaining_signers_value
+					local_var:true_neutral_applicant_signee_list_counter != imperia_true_neutral_remaining_signers_value
 				}
-				# If theres more remaining signers (on paper) than there are actual existing countries on the list, reduce the list by 1
-				change_variable = {
+				# If theres a different amount of remaining signers (on paper) than there are actual existing countries on the list, re-set it
+				set_variable = {
 					name = true_neutral_required_signers_var
-					subtract = 1
+					value = {
+						value = local_var:true_neutral_applicant_signee_list_counter
+						add = var:true_neutral_current_signers_var
```

Comment:
what? it's a reset right? why add this?

Reply:
Because the `true_neutral_required_signers_var` is the sum of the remaining AND current signers variables, or at least it should be.

Reply:
Wait are you telling me that if a country disappears, and it voted yes, that the vote doesn't get removed?

Reply:
I'll test that scenario

Reply:
Okay, so it looks like the annexed signer went into the `true_neutral_current_signers_var` pool, and as such the game doesn't see a difference between the signee_counter and the remaining_signers. So the `true_neutral_required_signers_var` variable is not re-set.

Reply:
whelp I don't quite understand it but it works so.....

Reply:
(though it is still limited by the monthly update xd)